lib/codebreaker/menu.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'game.rb'
+require_relative 'data_manager.rb'
+require_relative 'interface.rb'
+# comment
+class Menu
```

Comment:
Class is very similar to `Game.rb`. Do you use it for codebracker rack? If it is, please create some base class and then inherit from it.